### PR TITLE
Svelte 5 upgrade and fixed immediate breaking changes

### PR DESCRIPTION
**Pull request by stagrim**
Mostly solved form inside form issue by moving the dialog outside the outside form, and submit the form using the form attribute on the button elements

### CODE REVIEW

**Comment by stagrim** [comment]
Migrated to new format; hopefully everything works as well as it did previously now! We have a lot of illegal self closing tags though...

**Comment by stagrim** [comment]
Went through and manually fixed most lint issues. Svelte 5 seems to introduce many type errors related to props with components though. Don't know how to fix those

**Comment by stagrim** [comment]
Seems like I have managed to break the first page for those who are not logged in. I am not sure how to fix it

File: src/lib/components/forms/FormSubmitButton.svelte [hunk 1]
```diff
@@ -10,12 +10,14 @@
...
   let clazz: string | undefined = undefined;
   export { clazz as class };
+  export let form: string | undefined = undefined;
   const { delayed } = superform;
 </script>
 
 <LoadingButton
   class={clazz}
   type="submit"
+  {form}
```
Comment (Isak-Kallini) [inline src/lib/components/forms/FormSubmitButton.svelte:20]: What does this do?

**Review by Isak-Kallini — APPROVED** [review]
One small comment, looks good otherwise!

File: src/lib/components/forms/FormSubmitButton.svelte (hunk 1 above)
Comment (stagrim) [reply src/lib/components/forms/FormSubmitButton.svelte]: image

**Pull request merged**